packages/nodes-base/nodes/Xero/ContactDescription.ts

```diff
@@ -108,82 +108,82 @@ export const contactFields = [
 				default: '',
 				description: 'A user defined account number',
 			},
-			// {
-			// 	displayName: 'Addresses',
-			// 	name: 'addressesUi',
-			// 	type: 'fixedCollection',
-			// 	typeOptions: {
-			// 		multipleValues: true,
-			// 	},
-			// 	default: '',
-			// 	placeholder: 'Add Address',
-			// 	options: [
-			// 		{
-			// 			name: 'addressesValues',
-			// 			displayName: 'Address',
-			// 			values: [
-			// 				{
-			// 					displayName: 'Type',
-			// 					name: 'type',
-			// 					type: 'options',
-			// 					options: [
-			// 						{
-			// 							name: 'PO Box',
-			// 							value: 'POBOX',
-			// 						},
-			// 						{
-			// 							name: 'Street',
-			// 							value: 'STREET',
-			// 						},
-			// 					],
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'Line 1',
-			// 					name: 'line1',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'Line 2',
-			// 					name: 'line2',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'City',
-			// 					name: 'city',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'Region',
-			// 					name: 'region',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'Postal Code',
-			// 					name: 'postalCode',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'Country',
-			// 					name: 'country',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 				{
-			// 					displayName: 'Attention To',
-			// 					name: 'attentionTo',
-			// 					type: 'string',
-			// 					default: '',
-			// 				},
-			// 			],
-			// 		},
-			// 	],
-			// },
+			{
+				displayName: 'Addresses',
```

Comment:
You don't need these objects 2 times, you can just define it once and set the operation as create and update.

Reply:
@Jhalter5Stones These actually have to be 2 because they are inside of an additional fields option that has different options than each other so these can't be combined here.